components/match2/wikis/teamfortress/match_group_input_custom.lua

```diff
@@ -69,6 +69,7 @@ function CustomMatchGroupInput.processMatch(match, options)
 
 	match.stream = Streams.processStreams(match)
 	match.links = MatchFunctions.getLinks(match, games)
+	match.extradata = MatchFunctions.getExtraData(match)
```

Comment:
For future satefy, can you move this below `match.opponents`? See the PR that was recently merged